papers/Phi Fei Fo Fum (WIP)/paper.Rmd

```diff
@@ -0,0 +1,258 @@
+
+# Introduction
+
+Over the last two decades, there has been growing concerns about the so-called replication crisis in psychology and other fields [@OSC2015estimating; @camerer2018evaluating]. As a result, the scientific community has paid increasing attention to the issue of replicability in science, as well as to to good research and statistical practices.
+
+In this context, many have highlighted the limitations of null-hypothesis significance testing and called for more modern approaches to statistics [@cumming2014new]. One such recommendation coming for example from the New Statistics movement is to report effect sizes and their corresponding confidence intervals, and to increasingly rely on meta-analyses to increase confidence in those estimations. These recommendations are meant to complement (or even replace, according to some) null-hypothesis significance testing and would help transition toward a "cumulative quantitative discipline".
+
+These so-called "New Statistics" are synergistic because effect sizes are not only useful for interpreting study results in themselves, but also because they are necessary for meta-analyses, which aggregate effect sizes and their confidence intervals to create a summary effect size of its own [@lakens2013calculating].
+
+Unfortunately, popular software do not always offer the necessary implementations of the specialized effect sizes necessary for a given research design. In this paper, we review the most commonly used effect sizes for analyses of categorical variables that use the $\chi^2$ (chi-square) statistic, and introduce a new effect size---`r "\U05E4"` (Fei).
+
+Importantly, we offer researchers an applied walk-through on how to use these effect sizes in practice thanks to the `{effectsize}` package [@benshachar2020effectsize] in the R programming language [@base2023], which implements these measures and their confidence intervals. We cover in turn tests of independence (phi, Cramér’s *V*) and tests of goodness of fit (Cohen’s *w* and a new proposed effect size, Fei).
+
+# Tests of Independence
+
+<!-- Indra: Should we discuss differences between effect sizes for paired versus unpaired designs?  -->
+<!-- MSB: Hmmm... I'm not too familiar with paired tests - Are you talking about McNemar's test (and Cohen's g effect size)? -->
+
+The $\chi^2$ test of independence between two categorical variables examines if the frequency distribution of one of the variables is dependent on the other. 
```

Comment:
```suggestion
The $\chi^2$ test of independence between two categorical variables examines if the frequency distribution of one of the variables is dependent on the other. That is, are the two variables correlated such that, for example, members of group 1 on variable X are more likely to be members of group A on variable Y, rather than evenly spread across Y variable groups A and B. 
```